File: web3rpc/rpc-specs/code-samples/javascript/eth/transaction/getTransactionCount.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

  const address = "0xc94770007dda54cF92009BFF0dE90c06F603a09f";
  const blockNumberOrHashOrTag = "latest";

  provider.eth
    .getTransactionCount(
      address,
      blockNumberOrHashOrTag,
      {},
      (err, data, response) => {},
    )
    .then((data) => {
      console.log(data);
    });
})();

File: web3rpc/rpc-specs/code-samples/python/eth/transaction/getTransactionCount.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

address = "0xc94770007dda54cF92009BFF0dE90c06F603a09f"
blockTag = "latest"

w3 = Web3(Web3.HTTPProvider(host))
eth_response = w3.eth.get_transaction_count(address, blockTag)

print(eth_response)
